third-party/libhdfs3/Makefile

```diff
@@ -0,0 +1,35 @@
+ifndef CHPL_MAKE_HOME
+export CHPL_MAKE_HOME=$(shell pwd)/../..
+endif
+
+CHPL_MAKE_HOST_TARGET = --host
+include $(CHPL_MAKE_HOME)/make/Makefile.base
+include Makefile.share
+
+LIBHDFS3_FILE := libhdfs3/bootstrap
+
+default: $(LIBHDFS3_FILE)
+
+all: $(LIBHDFS3_FILE)
+
+clean: FORCE
+	rm -rf libhdfs3
+
+cleanall: FORCE
+	rm -rf build
+
+clobber: FORCE
+	rm -rf build install llvm
```

Comment:
right, side effect of a bad synch with origin/master - thought my backup copies (that had been tested) were correct - going back through and fixing them. apologies for the mess up (my git skills are primitive).